testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/other/DisableAutomaticRetriesTest.java

```diff
@@ -0,0 +1,52 @@
+package org.jboss.resteasy.test.client.other;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+
+/**
+ * Notify apache to disable its automatic retries.
+ * @tpSince RESTEasy 4.6.0
+ */
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class DisableAutomaticRetriesTest {
+
+    @Path("/test")
+    public static class  SimpleResource {
+        @GET
+        public String Hello() {
+            return "Hello";
+        }
+    }
+
+    @Deployment
+    public static Archive<?> deploy() {
+        WebArchive war = TestUtil.prepareArchive(
+                DisableAutomaticRetriesTest.class.getSimpleName());
+        return TestUtil.finishContainerPrepare(war, null, SimpleResource.class);
+    }
+
+    @Test
+    public void testFlag() throws Exception {
+        ResteasyClientBuilder builder = (ResteasyClientBuilder) ClientBuilder.newBuilder();
+        Assert.assertFalse(builder.isDisableAutomaticRetries());
+        builder.disableAutomaticRetries();
+        Client client = builder.build();
+        Assert.assertTrue(builder.isDisableAutomaticRetries());
+        client.close();
+    }
```

Comment:
This looks like a unit test more than an integration test. If that's what you really meant to do, please move this to the resteasy-client module test directory. If a real integration test is not too hard to write, it should live here, instead.

Reply:
done